docs/index.md

```diff
@@ -6,8 +6,12 @@ description: |-
   
 ---
 
-# scylladbcloud Provider
+# ScyllaDB Cloud Provider
 
+This provider allows you to manage [ScyllaDB Cloud](https://cloud.scylladb.com/) resources using Terraform.
+You must configure the provider with proper credentials before you can use it.
```

Comment:
Perhaps it's worth including this link https://cloud.docs.scylladb.com/stable/api-docs/api-get-started.html#obtaining-an-api-key-beta explaining how to obtain an API key

Reply:
Is that the token that should be used? If so, adding the link is an excellent idea.

Reply:
I think so 😅 might be worth verifying by @rjeczalik